Lib9c/Model/Mail/OrderExpirationMail.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Nekoyume.Model.Mail
 {
+    [Serializable]
     public class OrderExpirationMail : Mail
     {
         public readonly Guid OrderId;
```

Comment:
`public override MailType MailType => MailType.Auction;`
Please override the mail type